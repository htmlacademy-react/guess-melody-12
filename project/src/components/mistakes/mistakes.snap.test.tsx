import {render} from '@testing-library/react';
import Mistakes from './mistakes';

const count = 3;

describe('Component: Mistakes', () => {
  it('should render correctly', () => {
    const {container} = render(<Mistakes count={count} />);

    expect(container).toMatchSnapshot();
  });
});
