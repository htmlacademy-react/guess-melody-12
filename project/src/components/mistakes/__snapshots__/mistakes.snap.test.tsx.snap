// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Mistakes should render correctly 1`] = `
<div>
  <div
    class="game__mistakes"
  >
    <div
      class="wrong"
      data-testid="mistake"
    />
    <div
      class="wrong"
      data-testid="mistake"
    />
    <div
      class="wrong"
      data-testid="mistake"
    />
  </div>
</div>
`;
